include/z64actor.h

```diff
@@ -512,6 +512,46 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+// A set of 4 triangles which appear as a ring around an actor when the player Z-Targets it
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ f32 radius; // distance towards the center of the locked on actor
+    /* 0x10 */ Color_RGB8 color;
+} LockOnReticle; // size = 0x14
+
+typedef struct {
+    /* 0x00 */ Vec3f naviPos; // Used by Navi to indicate a targetable actor or general hint
+    /* 0x0C */ Vec3f lockOnPos;
+    /* 0x18 */ Color_RGBAf naviInnerColor;
+    /* 0x28 */ Color_RGBAf naviOuterColor;
+    /* 0x38 */ Actor* naviActor; // The actor that Navi hovers over
+    /* 0x3C */ Actor* lockOnActor;
+    /* 0x40 */ f32 naviMoveProgressFactor; // Controls Navi so she can smootly transition to the target actor
+    /* 0x44 */ f32 lockOnRadius; // Control the circle lock-on triangles coming in from offscreen when you first target
```

Comment:
```suggestion
    /* 0x44 */ f32 lockOnRadius; // Control the reticle coming in from offscreen when you first target
```
?

in general using "reticle" more, maybe in names too, sounds like a good idea, but I don't have a good enough understanding yet to make a suggestion

Reply:
This is an interesting effect where multiple reticles will be drawn on the screen at once to give the illusion of the reticle moving fast when initializing a lock-on, where a single reticle will contain 4 triangles (hence the `lockOnReticles[3]`). I don't think the idea that the this reticle is 4 triangles should be entirely removed but maybe it can just refer to reticle here

Reply:
Seems to me like reticle is fine here. high level it is controlling the radius of the reticle, even if underneath the effect is achieved with multiple reticles rendering in succession